scripts/kind-e2e/e2e.sh

```diff
@@ -194,7 +194,7 @@ function cleanup {
 function dump_clusters_on_error {
     for i in 1 2 3; do
         echo "Dumping cluster information for cluster$i ==========================="
-        kubectl --context=cluster$i cluster-info dump
+#        kubectl --context=cluster$i cluster-info dump
```

Comment:
Should either remove, remove comment, or move comment "#" to be indented (space on the left, not right).

Reply:
I didn’t mean to commit that ;-).